cmd/monitor/collector/bootstrap-agent.yaml

```diff
@@ -10,7 +10,7 @@ erda.oap.collector.core:
     metrics:
       - receivers:
           #- "erda.oap.collector.receiver.dummy"
-          - "erda.oap.collector.receiver.prometheus-remote-write"
+          - "erda.oap.collector.receiver.prometheus-remote-write@prometheus"
```

Comment:
`@prometheus` is meaningless. Maybe `@default` or other better name.

Reply:
adjusted, use `default`